test/models/confirmable_test.rb

```diff
@@ -215,10 +215,10 @@ def setup
       Devise.allow_unconfirmed_access_for = 5.days
       user = create_user
 
-      user.confirmation_sent_at = 4.days.ago
+      user.created_at = 4.days.ago
       assert user.active_for_authentication?
 
-      user.confirmation_sent_at = 5.days.ago
+      user.created_at = 5.days.ago
```

Comment:
I see an opportunity to refactor this test here. The expected behavior here is to check if the user is active to authentication as it was done in line 219. But below, we test a behavior when the user is not active for authentication. Wdyt about creating a separated test for this case?